lib/src/authenticate.tsx

```diff
@@ -111,6 +125,10 @@ const AuthProvider: FunctionComponent<PropsWithChildren<AuthProviderPropsInterfa
         return AuthClient.on(hook, callback);
     };
     const trySignInSilently = () => AuthClient.trySignInSilently(state, dispatch);
+    
+    const [ errors, setErrors ] = useState([]);
```

Comment:
Is it possible to type this from the exception type thrown from the low level SDK?
```suggestion
    const [ errors, setErrors ] = useState<AuthError[]>([]);
```